src/sezzle-checkout-button/README.md

```diff
@@ -10,10 +10,18 @@ The following are 4 methods for installing the Sezzle checkout button. The `Inst
 6. Overwrite the asset template with the code contents here[https://github.com/sezzle/static-widgets/blob/production/src/sezzle-checkout-button/sezzle-checkout-button-asset.js].
 7. Paste the following snippet in the bottom of the `layout/theme.liquid` file, then click Save:
 ```
-	{{ "sezzle-checkout-button.js" | asset_url | script_tag }}
-	<script>
-		new SezzleCheckoutButton({}).init();
-	</script>
+  {{ "sezzle-checkout-button.js" | asset_url | script_tag }}
+  <script>
+    const eventLogger = new EventLogger({
+      merchantUUID:"enter ID here" <!-- Your ID(format: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx) can be found in `business` section(under `settings`) of merchant dashboard -->
+    });
+    try{
+      new SezzleCheckoutButton({}).init();
+      eventLogger.sendEvent("checkout-button-onload")
```

Comment:
So, the `onload` event is registered only if the button has been loaded succesffully which makes sense. However, as far as I understand, the analytics team is more interested in how many merchants have the button active (installed in the themes and requires it to be loaded), rather than how many merchants actually have it successfully displayed on their cart page. @aholdahl please, correct me if I am wrong. 

In this case, the analytics has to sum up `onload` +`error` events to come up with the number of times the checkout button is being attempted to load. I am not sure if it's a problem (maybe not). If it is, I'd suggest logging `request` or `pre-load` or `pre-init` event before the `new SezzleCheckoutButton({}).init();` line which will represent the number of times the button is requested.

Reply:
@electrodrel Can we not compare the checkout button installed or not flag against the onload event to come up with that data?

Reply:
I am not sure we should rely on the checkout button installed flag for any analytics. That flag is used purely for the UI purposes  of those who install the button via UI. We have a bunch of merchants who have the button installed manually.

Reply:
even if we add `request` it is going to be combined with either `onload` or `error` so yeah from for analytics `onload`+`error` should do.

Reply:
I don't really oppose it, but would like to get @aholdahl feedback before resolving

Reply:
Well we definitely want to create distinction between whether a merchant has installed the successfully, or if they have installed it but unsuccessfully. The onload and error events should get us an accurate indication of that, so request or preload may be overkill - I suspect it's more likely to not receive events at all if the install is done wrong.